integration_tests/base_routes.py

```diff
@@ -830,6 +830,85 @@ def sync_router_di(request, router_dependencies):
     return router_dependencies["ROUTER_DEPENDENCY"]
 
 
+# ===== Split request body =====
+
+
+@app.get("/sync/split_request/query_params")
+def sync_split_request_basic(request: Request, query_params):
```

Comment:
@kliwongan would this work?
```suggestion
def sync_split_request_basic(query_params, request: Request):
```

Reply:
Yes it would. I run a search for a request object from the *args parameter, so it doesn't necessitate the request object being passed in first.

Reply:
@kliwongan , amazing. Could you add a test to assert that?